src/Library.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Automattic\WooCommerce\Blocks;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\AtomicBlock;
+use Automattic\WooCommerce\Blocks\Package;
+use Automattic\WooCommerce\Blocks\Assets\AssetDataRegistry;
+use Automattic\WooCommerce\Blocks\Assets\Api as AssetApi;
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationRegistry;
```

Comment:
probably don't need all these imports now?

Reply:
Ignore this, it was an artifact of the commits I was viewing and how the changes were being shown, I see they don't actually exist in the file.